test_api.py

```diff
@@ -0,0 +1,128 @@
+import requests
+import sqlite3
+
+from datetime import datetime
+# from pprint import pprint as pp
+
+
+DB = 'test.db'
+ID_G = iter(range(1, 10_000))
+
+init_data = [
+    (next(ID_G), 'Prepare a dinner for a week', 'home',
+     datetime.today().strftime('%d-%m-%y'), 0, ''),
```

Comment:
For time format '%d-%m-%y', you can use a constant variable, for easy supporting

Reply:
Fixed with TIME_FORMAT constant.